docs/development/standards.rst

```diff
@@ -35,70 +40,186 @@ Docker for builds
-    domain, so we can resolve all subdomains to ``localhost``. For example, here
-    are some related settings that affect project URLs::
-
-        PRODUCTION_DOMAIN = 'localhost:8000'
-        PUBLIC_DOMAIN = 'dev.readthedocs.io:8001'
-        USE_SUBDOMAIN = True
+    only be caught by using `USE_SUBDOMAIN` setting.
 
-Postgres as a database (optional for now)
+PostgreSQL as a database
     It is recommended that Postgres be used as the default database whenever
     possible, as SQLite has issues with our Django version and we use Postgres
     in production.  Differences between Postgres and SQLite should be masked for
     the most part however, as Django does abstract database procedures, and we
     don't do any Postgres-specific operations yet.
 
-    Because we run Postgres as our production database, you should be familiar
-    with maintenance of a Postgres instance however. You can run Postgres in a
-    Docker container on Linux or use the Postgres application on MacOS, both are
-    easy to get started with.
-
 Celery is isolated from database
-    Currently, it's really easy to forget that the Celery workers on our build
-    servers do not have database access and need to be written to use API access
-    instead. We don't yet have a pattern for this, but we should find a way to
-    put up a similar barrier in development, so that logic errors aren't
-    uncovered after we release.
+    Celery workers on our build servers do not have database access and need
+    to be written to use API access instead.
+
+Use NGINX as web server
+    All the site is served via NGINX with the ability to change some configuration locally.
+
+Azurite as Django storage backend
+    All static and media files are served using Azurite --an emulator of Azure Blob Storage,
+    which is the one used in production.
+
+Serve documentation via El Proxito
+    Documentation is proxied by NGINX to El Proxito and proxied back to NGINX to be served finally.
+    El Proxito is a small application put in front of the documentation to serve files
+    from the Django Storage Backend.
+
+Search enabled by default
+    Elasticsearch is properly configured and enabled by default.
+    Besides, all the documentation indexes are updated after a build is finished.
+
+
+Setup your environment
+----------------------
+
+After cloning ``readthedocs.org`` repository, you need to
+
+
+#. install `Docker <https://www.docker.com/>`_ following `their installation guide <https://docs.docker.com/install/>`_.
+
+#. install the requirements from ``common`` submodule:
+
+   .. prompt:: bash
+
+      pip install common/dockerfiles/requirements.txt
+
+#. build the Docker image for the servers:
+
+   .. prompt:: bash
+
+      inv docker.build
+
+#. pull down Docker images for the builders:
+
+   .. prompt:: bash
+
+      inv docker.pull
+
+#. start all the containers:
+
+   .. prompt:: bash
+
+      inv.docker up  --init
```

Comment:
```suggestion
      inv docker.up  --init  # --init is only needed the first time
```

Reply:
There should be some explanation here about exactly what this starts and how to access it (eg. http://community.dev.readthedocs.io/).